### ISSUE TITLE
Custom status indicator for incidents

### ISSUE THREAD

**Issue by jshorland** [ushahidi/platform#1824]
based on custom statuses associated with posts: "pending, open, in-progress, finished". 

Needs design on where to place indicator.

**Comment by justinscherer** [comment]
I covered the solution for this in #1825

**Comment by rjmackay** [comment]
I've added a basic version of this in https://github.com/ushahidi/platform-client-fix-our-environment/pull/1
Waiting on a branded pattern library before I try to go any further

**Comment by rjmackay** [comment]
Deployed!

**Comment by dalezak** [comment]
@rjmackay how was this implemented? Was it hardcoded in the web client? Or changed in the pattern library? Any idea how these changes will make it to the mobile app?

**Comment by rjmackay** [comment]
Its hardcoded in the web client.
- Because they've used several different surveys, status is a custom field on each survey.
- The client has a variable hardcoded in with the keys for each of those field (ie. attributes)
- It checks each of them when displaying the post card, and uses that to populate the status indicator
- For filtering I added a custom parameter to the posts API used like this `?giscon_status[]=Pending`. You can pass multiple values into that.

**Comment by dalezak** [comment]
I see, so there is a _Form_ _Attribute_ called `Incident Status` which has the the _Options_ ` "New", "In Progress", "Resolved"`, the value is displayed on the response card, plus is used in response filtering.
```
"id": 13,
"url": "https://fix-our-environment-api.ushahidi.com/api/v3/form_attributes/13",
"key": "e919c3a3-6a48-423a-8254-75e92bf86b76",
"label": "Incident Status",
"instructions": null,
"input": "select",
"type": "varchar",
"required": true,
"default": "New",
"priority": 0,
"options": [
    "New",
    "In Progress",
    "Resolved"
],
"cardinality": 1,
"config": [],
"form_stage_id": 4,
"response_private": false,
"allowed_privileges": [
    "read",
    "search"
]
```
Hmm, I don't see an easy way how we are gonna implement this in the mobile app 😟

**Comment by rjmackay** [comment]
2 be clear: there are 3 attributes called 'Incident Status', 1 per form.

**Comment by rjmackay** [comment]
This issue was moved to ushahidi/platform-fix-our-environment#21
